### PR TITLE
Update compare-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Made few changes to answer some common question about 

Which of the group types can be addressed by e-mail?
Which of the group types can be added to a Teams channel?
Which of the group types provide a shared calendar?

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4778

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/create-groups/compare-groups.md [hunk 1]
```diff
@@ -62,18 +64,24 @@ Security groups can contain users or devices. Creating a security group for devi
...
 
+Mail-enabled security groups can be added to a Teams channel.
+
 ## Shared mailboxes
 
 [Shared mailboxes](../email/create-a-shared-mailbox.md) are used when multiple people need access to the same mailbox, such as a company information or support email address, reception desk, or other function that might be shared by multiple people.
 
 Shared mailboxes can receive external emails if the administrator has enabled this.
 
+Shared mailbox includes a calendar that can be used for collaboration.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/compare-groups.md]:
```suggestion
Shared mailboxes include a calendar that can be used for collaboration.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**